tests/bsim/bluetooth/host/l2cap/multilink_peripheral/test_scripts/run.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Copyright (c) 2024 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+# [UK left the chat]
```

Comment:
not to be a grumpy old man, but this may be fun and all but is a bit of a waste of time..

Reply:
tests are always super-nitpicked anyways, it's my way of also poking fun at the reviewers. I'll of course remove those :)

Reply:
I'll try to remove those before PRs now, sorry about wasting your time